Colours_And_Fonts_style_v1.css

```diff
@@ -95,7 +161,7 @@ padding-top: 7px;
   
 /* Hover Colours*/
 a:link {color:#FFFFFF;} /* unvisited link */
-a:visited {color:#000000;} /* visited link */
+a:visited {color:#FFFFFF;} /* visited link */

```

Comment:
I changed the hover colour from black to white so end users can see the colours a lot better as black is too dark so it won't be as great for the end users eyes.